### PR TITLE
Create TernarySearch.java

### LINKED ISSUE #302: I want to add Ternary Search implementation in java
Please include a small description or summary of the issue here.

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Additional context**

**Languages**

-   [ ] Python
-   [x] Java : @Priyankarajput1
-   [ ] C
-   [ ] C++

### PR DESCRIPTION
**Pull request by Priyankarajput1**
Ternary Search in Java (#302 )

# Description

Please include a summary of the changes you have made in regard with this issue. 

Fixes #302

## Type of change

**Please delete options that are not relevant**

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] This change requires a documentation update

# Checklist:

-   [ ] I have read the contribution guidelines before making this PR.
-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented in my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation (Language folder or any other readme instructed to)
-   [ ] My changes generates no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: Java/TernarySearch.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Ternary search
```
Comment (kaustubhgupta) [inline Java/TernarySearch.java:1]: - Update java folder readme with your entry

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
- Fill the PR template properly

File: Java/TernarySearch.java (hunk 1 above)
Comment (Priyankarajput1) [reply Java/TernarySearch.java:1]: Updated the readme file. Kindly check, if any other changes required then let me know.

**Comment by kaustubhgupta** [comment]
@Priyankarajput1 PR template is not correctly filled

**Comment by kaustubhgupta** [comment]
We are not fools
https://www.geeksforgeeks.org/ternary-search/

**Pull request closed without merging**